Configs/en-de_ec.yml

```diff
@@ -25,7 +25,7 @@ TransferH: false # switch to hard gumbel after the model reaches target accuracy
 seed: 1
 mode: emergent_communication
 language_model_lambda: 0.125
-language_model_path: Output/mbart_lm_lr6e-6
+language_model_path: /projects/unmt/communication-translation/Models/mbart_lm_lr6e-6
```

Comment:
I don't think we should put full paths (as opposed to relative paths) in configs, so that they will be portable to the cloud

Reply:
Updated to reference the language_model_path via the DataLink symlink. One caveat here is that usages of this symlink (here and elsewhere) make this project trickier to export to GCP.

Reply:
Thanks!  Let's discuss this sym-link thing at our next meeting; don't think I've been aware of it.  Would it be better to just have everything as sub-dirs of the main repo, with judicious use of `.gitignore`?